src/components/ViewArticles.jsx

```diff
@@ -1,9 +1,33 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
 function ViewArticles() {
+  const [articles, setArticles] = useState([]);
```

Comment:
we also need a loading state to provide visual feedback to the user on slow load

Reply:
Done!!